crypto/elliptic-curve-crypto/src/lib.rs

```diff
@@ -198,10 +196,12 @@ mod tests {
         assert!(verify(&digest, &signature, &public_key));
     }
 
-    #[quickcheck]
-    fn test_ecdsa(digest: ScalarFieldElement, private_key: ScalarFieldElement) -> bool {
-        let public_key = private_to_public(&private_key);
-        let signature = sign(&digest, &private_key);
-        verify(&digest, &signature, &public_key)
-    }
+    proptest!(
+        #[test]
+        fn test_ecdsa(digest: ScalarFieldElement, private_key: ScalarFieldElement) {
+            let public_key = private_to_public(&private_key);
+            let signature = sign(&digest, &private_key);
+            prop_assert!(verify(&digest, &signature, &public_key));
+        }
+    );
```

Comment:
How do we feel about this vs using the closure notation for proptest?

Reply:
I like the closure notation. I'm going to switch everything over in another PR.